src/axom/quest/interface/inout.cpp

```diff
@@ -19,6 +19,23 @@ namespace quest
 {
 namespace internal
 {
+/// Parameter variables for the InOutQuery
+struct InOutParameters
+{
+  bool m_verbose {false};
+  int m_dimension {3};
+  int m_segmentsPerKnotSpan {25};  /// Used when linearizing curves
+  double m_vertexWeldThreshold {1E-9};
+
+  void setDefault()
+  {
+    m_verbose = false;
```

Comment:
To avoid repeating values, you could do `*this = InOutParameters{};`

Reply:
Great suggestion!